src/index.js

```diff
@@ -541,6 +547,23 @@ function MarkdownEditor(props) {
     event.preventDefault();
     return false;
   });
+  /**
+   * Render the static-editing toolbar.
+   */
+  const renderEditor = useCallback((props, editor, next) => {
+    const children = next();
```

Comment:
what does rendering `next()` as the children do here? I don't fully understand this part.

Reply:
[This](https://docs.slatejs.org/guides/rendering#the-editor-itself) would be the best I can do on a source for this.

Reply:
Got it, thank you!